docker/owlbot/nodejs/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+# Doesn't get sourced in root shell, for reasons I don't understand.
+source "$HOME/.bashrc"
+set -x
+cd "$OWLBOT_WORKING_DIR"
```

Comment:
Currently we run the container like this:

```
  - name: ${_CONTAINER}
    dir: '${_REPOSITORY}'
```

It sounds like we'd want to se an additional environment variable equal to `OWLBOT_WORKING_DIR`?

Reply:
Actually, we can run this container the same way:

```yaml
  - name: ${_CONTAINER}
    dir: '${_REPOSITORY}'
```

because `$OWLBOT_WORKING_DIR` will be empty, and calling `cd` with no arguments yields no error and no change in directory.

But that triggers the question do we need OWLBOT_WORKING_DIR at all?  I suppose not.  I'll remove it.